lib.php

```diff
@@ -8495,6 +8495,11 @@ function hsuforum_view($forum, $course, $cm, $context) {
  * @since Moodle 2.9
  */
 function hsuforum_discussion_view($modcontext, $forum, $discussion) {
+    $pageWasRefreshed = isset($_SERVER['HTTP_CACHE_CONTROL']) && $_SERVER['HTTP_CACHE_CONTROL'] === 'max-age=0';
```

Comment:
Not sure where this is coming from but this code I think did not work on safari

Reply:
That was part of the original code that Rumano did, not too sure about that

Reply:
I would suggest pulling in origin master to check if this code is still there or removed.

Reply:
Looks like it is removed in master. Removing now